### PR TITLE
LINTING ERB, JS, SCSS

### PR DESCRIPTION
**Pull request by paul-mesnilgrente**

### CODE REVIEW

File: app/assets/stylesheets/blocks/_details.scss [hunk 1]
```diff
@@ -13,12 +13,16 @@
...
+        background-color: scale-color(
+          $color,
+          $lightness: $callout-background-fade
+        );
+        padding: 0.5rem 1rem;
       }
 
       .details__content {
-        border: 2px solid scale-color($color, $lightness: $callout-background-fade);
+        border: 2px solid
```
Comment (EdwinRozario) [inline app/assets/stylesheets/blocks/_details.scss:24]: lets try using `rem` everywhere for consistency?

File: app/assets/stylesheets/blocks/_forms.scss [hunk 2]
```diff
@@ -89,15 +91,16 @@ fieldset#management_permissions , fieldset#harvesting_permissions {
   padding-bottom: 10px;
```
Comment (EdwinRozario) [inline app/assets/stylesheets/blocks/_forms.scss:91]: Same here.

File: app/assets/stylesheets/blocks/_forms.scss [hunk 3]
```diff
@@ -89,15 +91,16 @@ fieldset#management_permissions , fieldset#harvesting_permissions {
...
-  padding:0!important;
+  border: none;
+  padding: 0 !important;
   font: inherit;
   cursor: pointer;
 }
 
 .resume {
-  color: #FA5858;
+  color: #fa5858;
```
Comment (EdwinRozario) [inline app/assets/stylesheets/blocks/_forms.scss:105]: Should we have a file for colours so they are a bit more organised?

File: app/assets/stylesheets/blocks/_home.scss [hunk 4]
```diff
@@ -29,9 +28,6 @@
       &.stat {
         font-size: 27px;
```
Comment (EdwinRozario) [inline app/assets/stylesheets/blocks/_home.scss:29]: rem

File: app/assets/stylesheets/blocks/_parsers.scss [hunk 5]
```diff
@@ -112,7 +112,8 @@ $arrowSize: 18px;
 #parser-versions {
   margin-bottom: 20px;
```
Comment (EdwinRozario) [inline app/assets/stylesheets/blocks/_parsers.scss:113]: rem

**Review by EdwinRozario — APPROVED** [review]
Good job cleaning all this up. I only have 2 suggestions.
- Use rem everywhere
- Move all html colour codes to a file and name them

**Comment by paul-mesnilgrente** [comment]
Your suggestions will be implemented in follow up stories

**Pull request merged**